src/main/features/claim/views/macro/about-claim-widget.njk

```diff
@@ -15,7 +17,7 @@
   <ul class="font-xsmall list">
     <li>
       <strong class="bold">{{ t(otherPartyLabel) }}</strong>
-      <div>{{ otherPartyName }}</div>
+      <a class="block" {{ internalLink(t(otherPartyName), DashboardPaths.contactThemPage.evaluateUri({ externalId: claim.externalId })) }}</a>
```

Comment:
this looks like a link inside a link? perhaps a div instead of an a?

Reply:
Changed that to a div 👍